x-pack/legacy/plugins/ingest/server/libs/policy.ts

```diff
@@ -14,10 +13,10 @@ import { PolicyAdapter } from './adapters/policy/default';
 import { DatasourcesLib } from './datasources';
 import { BackendFrameworkLib } from './framework';
 import { OutputsLib } from './outputs';
-import { Status, Datasource, Policy } from './types';
+import { Status, Datasource, Policy, PolicyUpdateHandler } from './types';
 
 export class PolicyLib {
-  public events: EventEmitter = new EventEmitter();
+  public eventsHandler: PolicyUpdateHandler[] = [];
```

Comment:
do we expect multiple event handlers to be potentially registered?

Reply:
I do not think